### ISSUE TITLE
Share Your Games FEATURE ♘

### ISSUE THREAD

**Issue by vatan45** [Bot-Rakshit/bm_frontend#81]
We need to implement a new page titled "Share Your Games" that allows users to share and view chess games. This page should include the ability for users to post their games, view games shared by others, and interact with them. The feature will enhance user engagement by allowing users to showcase their chess games directly on our platform.

Functionality Requirements
1) posting a game
2) game feed
3) upvote functionality
4) comment section

**Comment by Sparsh-Malhotra** [comment]
I am picking up games listing page

**Comment by RitikBora** [comment]
Hey guys,

I found this issue open for quite some time and thought of contributing.

As of now I am able to fetch recent games for a user and user can select which game needs to be shared. 
Sharing a short video clip of it.

Kindly share your feedback on this @vatan45 @Bot-Rakshit  . Im happy to discuss further steps.


https://github.com/user-attachments/assets/4f1216bf-3673-4ab9-8039-66957f000aae

Thanks,
Ritik

**Comment by RitikBora** [comment]
Yes , Im fetching from the backend itself. Both frontend and backend are running on my local. Backend is the one calling chess.com API's . The chess routes section.

I forked your backend as well.  I can add those changes to the turborepo as well.

**Comment by RitikBora** [comment]
Hey @Bot-Rakshit ,
Is the turbo repo private a of now?

**Comment by RitikBora** [comment]
Cool looking forward to contributing!
